packages/react/src/global-nav-profile-menu/GlobalNavProfileMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ReactNode } from "react";
+
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { IconEllipsis } from "../icons/IconEllipsis";
+import { Text } from "../text";
+import * as styles from "./GlobalNavProfileMenu.css";
+
+export type GlobalNavProfileMenuProps = BoxProps<
+  "div",
+  {
+    avatar: ReactNode;
+    name?: string;
+    organization?: string;
```

Comment:
What if we handle them through children like the way we handled label? So, we just wrap the children with user-Info styling, and they can have their own style or related information within the children?